lib/rack/headers_filter.rb

```diff
@@ -0,0 +1,38 @@
+module Rack
```

Comment:
maybe we can nod to the gem in a comment here

Reply:
Yup, we probably should, good call!